knot_sim/protocol_core/modbus/load_config.py

```diff
@@ -1,7 +1,7 @@
 import json
 import sys
 
-from protocol_core import defines as defs
+from knot_sim.protocol_core import defines as defs
```

Comment:
Have you considered naming the folder `simulator`? Just a suggestion :smile: 

```suggestion
from simulator.protocol_core import defines as defs
```

Reply:
I believe that all the changes related to moving the files to the "knot_sim" folder and adapting the modules to support this change should be in a separated commit.

Reply:
done